test/cppsuite/test_harness/timestamp_manager.h

```diff
@@ -59,6 +59,9 @@ class timestamp_manager : public component {
     /* Get a unique timestamp. */
     wt_timestamp_t get_next_ts();
 
+    /* Generate a random timestamp between the oldest timestamp and now. */
+    wt_timestamp_t get_random_ts();
```

Comment:
Alphabetize and you can add `const` at the end of the declaration (and impl) as it does not modify any class member.